<?xml version="1.0" encoding="UTF-8" ?>
<testsuites errors="1"
            failures="0"
            tests="1">

  <testsuite name="ValidateIds"
    errors="1" failures="0" hostname="" id="" package="ValidateIds" skipped="0"
    tests="1" time="0.008000" timestamp="Wed, 16 Jan 2019 21:55:55 GMT">
  
    <testcase name="QOBB-75 Check for valid ID" classname="ValidateIds" time="0.008000" assertions="0">
    </testcase>
  

  
    <system-err>
        Error while running [Validate Ids / QOBB-75 Check for valid ID]:

  ReferenceError: numElems is not defined
    at Object.QOBB-75 Check for valid ID (C:\Users\bethl_000\Documents\Beth\Learning\DevMountain\EmployeeManager-V2\tests\ValidateIds.js:25:29)
    at Module.call (C:\Users\bethl_000\AppData\Roaming\npm\node_modules\nightwatch\lib\runner\module.js:62:34)
    at C:\Users\bethl_000\AppData\Roaming\npm\node_modules\nightwatch\lib\runner\testcase.js:70:29
    at _fulfilled (C:\Users\bethl_000\AppData\Roaming\npm\node_modules\nightwatch\node_modules\q\q.js:834:54)
    at C:\Users\bethl_000\AppData\Roaming\npm\node_modules\nightwatch\node_modules\q\q.js:863:30
    at Promise.promise.promiseDispatch (C:\Users\bethl_000\AppData\Roaming\npm\node_modules\nightwatch\node_modules\q\q.js:796:13)
    at C:\Users\bethl_000\AppData\Roaming\npm\node_modules\nightwatch\node_modules\q\q.js:556:49
    at runSingle (C:\Users\bethl_000\AppData\Roaming\npm\node_modules\nightwatch\node_modules\q\q.js:137:13)
    at flush (C:\Users\bethl_000\AppData\Roaming\npm\node_modules\nightwatch\node_modules\q\q.js:125:13)
    at process._tickCallback (internal/process/next_tick.js:61:11)
    </system-err>
  

  
  </testsuite>
</testsuites>
